driver/pace/driver/driver.py

```diff
@@ -273,6 +273,9 @@ def from_dict(cls, kwargs: Dict[str, Any]) -> "DriverConfig":
             kwargs["grid_config"] = GridInitializerSelector.from_dict(
                 kwargs["grid_config"]
             )
+            grid_type = kwargs["grid_config"].config.grid_type
+            if grid_type != 0:
```

Comment:
Can we get some ENUM instead of `grid_type != 0`? I'd rather have a `grid_type in [A, B, C]` make it explicit. This comment is applicable to a lot of other areas where we should switch the integer for a readable const/enum

Reply:
In general I agree, but since `grid_type = 0` is the default I just wanted to make sure any other grid type gets copied over to the dycore config correctly

Reply:
Comment to that effect then, so that it ever change we know to make it strictier